doc/vim-lsp.txt

```diff
@@ -680,6 +681,28 @@ g:lsp_diagnostics_virtual_text_prefix   *g:lsp_diagnostics_virtual_text_prefix*
 	let g:lsp_diagnostics_virtual_text_prefix = "> "
 	let g:lsp_diagnostics_virtual_text_prefix = " ‣ "
 
+g:lsp_linked_editing_enabled                  *g:lsp_linked_editing_enabled*
+    Type: |Number|
+    Default: `1`
+
+    Enable textDocument/linkedEditingRange feature.
+
+    NOTE: If you using `vim-matchup`, recommend to set `g:matchup_matchparen_deferred = 1`
+
+    Example: >
+	let g:lsp_linked_editing_enabled = 1
+	let g:lsp_linked_editing_enabled = 0
+
+	" The below mappings enable to support `d` or `c` commands.
+	nnoremap <buffer> <Plug>(vimrc-d) d
+	xnoremap <buffer> <Plug>(vimrc-d) d
+	nnoremap <buffer> <Plug>(vimrc-c) c
+	xnoremap <buffer> <Plug>(vimrc-c) c
+	nmap <buffer> d <Plug>(lsp-linked-editing-range-prepare)<Plug>(vimrc-d)
+	xmap <buffer> d <Plug>(lsp-linked-editing-range-prepare)<Plug>(vimrc-d)
+	nmap <buffer> c <Plug>(lsp-linked-editing-range-prepare)<Plug>(vimrc-c)
+	xmap <buffer> c <Plug>(lsp-linked-editing-range-prepare)<Plug>(vimrc-c)
```

Comment:
Sorry. I noticed it does not work.

But "nnoremap <buffer><expr> d lsp#internal#linked_editing_range#prepare() . 'd'" is works fine.

I don't know how to expose mapping function in vim-lsp. (`lsp#mapping#linked_editing_range_prepare()?`)